### DISCUSSION TITLE
Welcome to facebook_crawler Discussions!

### DISCUSSION THREAD

**Discussion by tlyu0419** [TLYu0419/facebook_crawler#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jiechao15** [comment]
Hello,

I am using your parkage. It is the most easiest one I saw. After running, it's always showing the following message as below. How can I know when the program is done? and after it is done, where can I find the excel file?

"The maximum date of these posts is: 2022-10-11, keep crawling..."

**Reply by tlyu0419** [reply]
Please adjust the until_date parameter, and this will do.
